src/middlewared/middlewared/etc_files/syslog-ng/conf.d/tnaudit.conf.mako

```diff
@@ -43,7 +44,7 @@
 
     def get_db(svc):
         sql = 'sql(type(sqlite3)'
-        db = f'database("{os.path.join(AUDIT_DIR, svc)}.db")'
+        db = f'database("{audit_file_path(svc)}")'
         table = 'table("audit_${TNAUDIT.svc}_${TNAUDIT.vers.major}_${TNAUDIT.vers.minor}")'
```

Comment:
Why do we distinguish versions in table names? Do we plan to copy audit logs to new BE on upgrade?

Reply:
At present there are no plans to copy them forward between versions. The idea being that local audit records are somewhat ephemeral and if someone is really serious about auditing they should configure to send them off remotely (TODO). But plans can always change. By putting them in separate table we avoid having syslog-ng try to automatically create new columns in the DB if we decide to change schema (indicated by version bump).